sklearn/linear_model/ridge.py

```diff
@@ -1586,14 +1586,17 @@ class RidgeCV(_BaseRidgeCV, RegressorMixin):
 
     Parameters
     ----------
-    alphas : numpy array of shape [n_alphas]
+    alphas : numpy array of shape [n_alphas], list or tuple
         Array of alpha values to try.
         Regularization strength; must be a positive float. Regularization
         improves the conditioning of the problem and reduces the variance of
         the estimates. Larger values specify stronger regularization.
         Alpha corresponds to ``C^-1`` in other linear models such as
         LogisticRegression or LinearSVC.
+        Default value is (0.1, 1.0, 10.0)
         If using generalized cross-validation, alphas must be positive.
+        Note: the `alphas` feature will be of `np.ndarray` type even if the
+        constructor parameter was a list or a tuple.
```

Comment:
Also, I would remove this. As mentioned above it's generally true for most parameters / attributes.